Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -1081,7 +1081,7 @@ void VideoConfigDiag::PopulatePostProcessingShaders()
   if (shaders.empty())
     return;
 
-  choice_ppshader->AppendString(_("(off)"));
+  choice_ppshader->AppendString(_("Off"));
```

Comment:
I'm not sure. The shaders we ship with use lowercase names, so writing this in uppercase makes it look a bit out of place. But on the other hand, maybe it actually is good that it looks out of place so it's easier to notice that it's a special option.

You have a point about Stereoscopic 3D Mode using Off. I was only thinking of how things looked within one dropdown, not how the whole tab looks when you're not inside a dropdown.